spec/index.bs

```diff
@@ -6020,6 +6016,15 @@ type and each [=binding type=] has an associated [=internal usage=], given by th
 
                     Note: See {{GPUExternalTexture}} for an explanation of this behavior.
             </dl>
+
+    <div class=compatmode>
+        3. If device.{{device/[[features]]}} does not [=list/contain=]
```

Comment:
nit: `device` should be a variable, `|device|` (but see other comment before doing this)

Reply:
Ack